src/chart/cartesianChart.js

```diff
@@ -19,32 +19,53 @@ export default function(xScale, yScale) {
         },
         yLabel = '',
         xLabel = '',
+        xBaseline = noop,
```

Comment:
Actually thinking again about it, `null` seems a more consistent (with the rest of d3) value. Maybe it could be special cased in the property instead?